pkg/helm/actions/config.go

```diff
@@ -11,6 +11,7 @@ import (
 )
 
 var settings = cli.New()
+var k8sInClusterCA = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
```

Comment:
This is incorrect, we have flags for passing certs/CA already:
https://github.com/openshift/console/blob/master/cmd/bridge/main.go#L91
and you can see how the operator passes the service CA via a flag when it deploys the console:
https://github.com/openshift/console-operator/blob/master/pkg/console/subresource/deployment/deployment.go#L279
You can `oc get deployment console` on a running cluster if you want to see this.

Reply:
using incluster config now